src/pages/publication/EditPublication.tsx

```diff
@@ -73,9 +74,14 @@ const EditPublication: FC = () => {
     setShowForm(true);
   };
 
+  const handleTemporaryButtonClick = () => {
+    setShowForm(true);
+    history.push('/publication/new');
```

Comment:
vi har ingen route for `/publication/new` i AppRoutes, bruker bare `/publication`

Reply:
samme kommentar som over ☝️ sjekk navnet til funksjonen

Reply:
kan skrive noe annet enn `/new` også da, men vi må vel uansett snart fjerne den knappen (og skrive om alle testene 🙃 )